test/clt-tests/sharding/base/init.recb

```diff
@@ -0,0 +1,27 @@
+––– input –––
+set -b
+––– output –––
+––– input –––
+apt-get update -y > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+apt-get install -y git > /dev/null; echo $?
+––– output –––
+0
+––– input –––
+git clone -b sharding/inserts -q https://github.com/manticoresoftware/manticoresearch-buddy.git /workdir; echo $?
```

Comment:
This way when we need custom buddy only for inserts, we can use to start daemon with EXTRA_SECTION=... so we change in one place in that case

Reply:
Discussed this, waiting for a fix and then I'll remove the init block.